root/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Views/Crud1/Index.cshtml

```diff
@@ -0,0 +1,118 @@
+﻿@*
+    //**********************************************************************************
+    //* サンプル アプリ・モデル
+    //**********************************************************************************
+
+    // テスト用クラスなので、必要に応じて流用 or 削除して下さい。
+
+    //**********************************************************************************
+    //* クラス名        ：Index
+    //* クラス日本語名  ：CRUDサンプル（Html.BeginForm）
+    //*
+    //* 作成日時        ：－
+    //* 作成者          ：－
+    //* 更新履歴        ：－
+    //*
+    //*  日時        更新者            内容
+    //*  ----------  ----------------  -------------------------------------------------
+    //*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
+    //**********************************************************************************
+*@
+
+@using MVC_Sample.Models.ViewModels
+@model CrudViweModel
+
+@{
+    // ページタイトル
+    ViewBag.Title = "CRUDサンプル（Html.BeginForm）";
+
+    // マスターページ
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@* スクリプト・セクション（RenderSection("HeaderScriptsSection", required: true)に出力されます） *@
+@section HeaderScriptsSection{
+    <!-- Head 部の section -->
+}
+
+@* 本文レイアウト（RenderBodyに出力されます） *@
+@* Html.BeginFormは従来の全体方式、Ajax.BeginFormは部分更新方式（UpdateTargetIdの範囲を部分更新） *@
+@using (Html.BeginForm(null, null, FormMethod.Post, new { id = "form1", onsubmit = "Fx_OnSubmit();" }))
+{
+    @Html.AntiForgeryToken()
+
+    <div class="form-inline">
+        <div class="form-group">
```

Comment:
@daisukenishino 

> - Please point url of an accurate description( e.g. official reference etc ) of the following classes.
> ```XML
> <div class="form-group">
> ```

Please refer [Bootstrap official site](http://getbootstrap.com/css/#forms).

> - I think that the following instructs the operation of the grid system.  
> Please tell me the contents of this instruction.
> ```XML
> <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
> ```

Please refer [Bootstrap official site](http://getbootstrap.com/css/#grid).

A half width (i.e. 6 columns of 12 columns) of the browser is secured by writing as `<div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">`.

> - Is the following unnecessary for the div tag surrounding the Button?
> ```XML
> <div class="row">
> ```

Grid systems are used in `<div class="row">`.

If grid systems are not used, *row* class is unnecessary.

> - I found a description that not only form-group and form-control but also control-label class or "for" attribute is necessary. Do you think that it is better to give control-label class or "for" attribute ?

Please refer [Bootstrap official site](http://getbootstrap.com/css/#forms).

*control-label* class is optional.
*control-label* class seems to be used with *form-horizontal*, *has-success*, *has-warning*, *has-error* classes.

*for* attribute is used for associating label and input control.
Please refer [this page](http://www.newcredge.com/IT/www/html/tag/form/label.html).